### ISSUE TITLE
[Bug]:  Cannot Find VAE on startup of WebUI

### ISSUE THREAD

**Issue by rushuna86** [anapnoe/stable-diffusion-webui-ux#146]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

Every start up with WebUI console showing "Couldn't find VAE named XXXXX" Using None Instead.

### Steps to reproduce the problem

Every Start Up of WebUI

### What should have happened?

Load selected VAE from SD_VAE.

### Commit where the problem happens

Update 1.2.0 (f2b9c2c was working fine)

### What platforms do you use to access the UI ?

Windows

### What browsers do you use to access the UI ?

Google Chrome

### Command Line Arguments

```Shell
--opt-sdp-attention --no-half-vae --autolaunch
```

### List of extensions

Built-in only. Happens with clean install as well.

### Console logs

```Shell
Couldn't find VAE named clearvae_variant.safetensors; using None instead
Applying scaled dot product cross attention optimization.
Textual inversion embeddings loaded(7): bad_prompt_version2-neg, badhandv4, badv5, easynegative, ng_deepnegative_v1_75t, pureerosface_v1, ulzzang-6500-v1.1
Model loaded in 2.1s (load weights from disk: 0.4s, create model: 0.2s, apply weights to model: 0.8s, apply half(): 0.3s, move model to device: 0.4s).
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████████| 86/86 [00:00<00:00, 844.86it/s]
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████████| 86/86 [00:00<00:00, 828.78it/s]
ControlNet v1.1.173
ControlNet v1.1.173
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 8.4s (import torch: 0.9s, import gradio: 0.6s, import ldm: 0.3s, other imports: 1.4s, load scripts: 3.4s, create ui: 1.6s, gradio launch: 0.1s).
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:02<00:00,  8.46it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:00<00:00, 31.17it/s]
Loading VAE weights specified in settings: D:\SD\WebUIX\models\VAE\ClearVAE_NansLess1.safetensors0:00<00:00, 35.61it/s]
Applying scaled dot product cross attention optimization.
VAE weights loaded.
Loading VAE weights specified in settings: D:\SD\WebUIX\models\VAE\clearvae_variant.safetensors
Applying scaled dot product cross attention optimization.
VAE weights loaded.
```

### Additional information

After WebUI Loads inside VAE List quicksetting actually shows clearvae_variant.safetensors as selected. Inside settings also shows it as selected, but image generation is done with None as per console. Have to go in quicksetting/setting change to different VAE and back for it to load.

**Comment by rushuna86** [comment]
**Console from Commit f2b9c2c with exactly the same everything even all extensions loaded**
Launching Web UI with arguments: --opt-sdp-attention --no-half-vae --autolaunch
No module 'xformers'. Proceeding without it.
Civitai Helper: Get Custom Model Folder
Civitai Helper: Load setting from: D:\SD\WebUIX\extensions\Stable-Diffusion-Webui-Civitai-Helper\setting.json
Civitai Helper: No setting file, use default
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████████| 86/86 [00:00<00:00, 842.79it/s]
[AddNet] Updating model hashes...
100%|█████████████████████████████████████████████████████████████████████████████████| 86/86 [00:00<00:00, 844.23it/s]
ControlNet v1.1.173
ControlNet v1.1.173
Loading weights [db1b218a5b] from D:\SD\WebUIX\models\Stable-diffusion\Chibi-Mix.safetensors
Creating model from config: D:\SD\WebUIX\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: D:\SD\WebUIX\models\VAE\clearvae_variant.safetensors
Applying scaled dot product cross attention optimization.
Textual inversion embeddings loaded(7): bad_prompt_version2-neg, badhandv4, badv5, easynegative, ng_deepnegative_v1_75t, pureerosface_v1, ulzzang-6500-v1.1
Model loaded in 1.9s (load weights from disk: 0.3s, create model: 0.2s, apply weights to model: 0.6s, apply half(): 0.3s, move model to device: 0.3s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 8.4s (import torch: 0.9s, import gradio: 0.6s, import ldm: 0.3s, other imports: 1.6s, load scripts: 1.1s, load SD checkpoint: 2.2s, create ui: 1.4s, gradio launch: 0.3s).

**Comment by anapnoe** [comment]
isn't the default SD VAE Automatic?

**Comment by rushuna86** [comment]
yes updated to the last commit. And Default VAE is automatic, but if you go into settings>Stable Diffusion and change the VAE in there to something else. It will save that VAE and will load that one up when you start up WebUI. Like it does in console,
Loading VAE weights specified in settings: D:\SD\WebUIX\models\VAE\clearvae_variant.safetensors

But since 1.2.0 update console always showing it cannot find VAE named (selected vae), but inside settings/quick setting the named VAE is actually shows as the selected VAE, but isn't actually applied. Have to change to different vae and back for it to be applied.

**Comment by anapnoe** [comment]
you shouldn't have 2 SD VAE options there should be only one in settings or quicksettings can you copy your settings to another location and restart ui-ux and add it to quicksettings and try again the new update from upstream mess the quicksettings

**Comment by rushuna86** [comment]
still the same. https://imgur.com/tx7Fi6D

This issue isnt because of the recent commits. It's been happening since b08500c

**Comment by anapnoe** [comment]
![Untitled](https://github.com/anapnoe/stable-diffusion-webui-ux/assets/124302297/433852d0-de91-45c6-a7d8-211dcb0fd07f)

something is not ok I cant reproduce the issue
does your SD VAE show in settings and quicksettings

**Comment by rushuna86** [comment]
Nope it shows in either one. And yep something is really messed up. I did a brand new pull and another clean install, used my old config.json and it works O_o so weird. didnt work before. i'll try figure it out and if i find out what caused it i'll let u know

**Comment by rushuna86** [comment]
@anapnoe https://github.com/vladmandic/sd-extension-system-info
